### ISSUE TITLE
Lens comparison

### ISSUE THREAD

**Issue by lazd** [lacymorrow/fpv-bible#7]
Side-by-side outdoors comparison of tons of different kinds of lenses. There has been no really good comparison.

We could also do a side-by-side mounting of two cameras with similar lenses (say 2.1 and 1.8, then a 2.5 and 2.8) on an actual quad with two VTXs and show the DVR of a freestyle run.

**Comment by lazd** [comment]
I sort of tried to do basic photos here with a camera mounted in a vise, but it doesn't do justice: http://imgur.com/a/ZYFU9

**Comment by lacymorrow** [comment]
yes, seth phreakin also posted some settings we should test in different lights and such.
This is such an easy one to do!

**Comment by lacymorrow** [comment]
here's Phreakin's: https://imgur.com/a/yR0NQ

**Comment by lacymorrow** [comment]
📋 Reviewed by Paperclip — this repo is legacy/unmaintained. No internal task created.

**Comment by lacymorrow** [comment]
Tracked in Paperclip as [LAC-1192](https://paperclip.ing/LAC/issues/LAC-1192) — stale issue audit for legacy repos.

**Comment by lacymorrow** [comment]
Closing — this repo is no longer actively maintained and this content idea is outdated. Thank you for the contribution.
